files/initramfs-tools/union-mount.j2

```diff
@@ -6,17 +6,17 @@ case $1 in
 esac
 
 ## Mount the aufs file system: rw layer over squashfs
-mkdir -p ${rootmnt}/host/rw
-mount -n -o dirs=${rootmnt}/host/rw:${rootmnt}=ro -t aufs root-aufs ${rootmnt}
+mkdir -p ${rootmnt}/host/image-%%GIT_REVISION%%/rw
+mount -n -o dirs=${rootmnt}/host/image-%%GIT_REVISION%%/rw:${rootmnt}=ro -t aufs root-aufs ${rootmnt}
 ## Check if the root block device is still there
 [ -b ${ROOT} ] || mdev -s
 ## Mount the raw partition again
 mount ${ROOT} ${rootmnt}/host
 ## Mount the working directory of docker engine in the raw partition, bypass the aufs
 mkdir -p ${rootmnt}/var/lib/docker
-mount --bind ${rootmnt}/host/{{ DOCKERFS_DIR }} ${rootmnt}/var/lib/docker
+mount --bind ${rootmnt}/host/image-%%GIT_REVISION%%/{{ DOCKERFS_DIR }} ${rootmnt}/var/lib/docker
 ## Mount the boot directory in the raw partition, bypass the aufs
```

Comment:
I do not need this commit to support sonic-to-sonic on arista aboot platform, but it is better not to break the existing func.